### ISSUE TITLE
Latch open (and buzzer on) time appears to be inconsistent

### ISSUE THREAD

**Issue by C-o-r-E** [heliosmakerspace/accessctl#1]
As the title says.

**Comment by Alex-II** [comment]
How inconsistent? It should be +/- 1 second to whatever it's set

**Comment by C-o-r-E** [comment]
Extremely. Sometimes the duration lasts for less than a second (subjective estimate).

I first observed it before touching the code. Since then I modified the delay mechanism to simply use a time.sleep() call and the problem persists.

It could be a hardware problem. More testing required.

**Comment by Alex-II** [comment]
I think that when we were testing it with a 15 seconds door-open-time 1 or
2 weeks ago, the actual door-open-time was between 14 and 16 seconds.
I remember looking at the code and thinking this delay was caused by
long-ass functions calls to the lib that queries the RFID sensor. But maybe
not.
In any case, if there is indeed a +/- 1 second error (as I remember from
the tests last time), then setting the door-open-time to 1.5 seconds will
product variations and make it open between 0.5 and 2.5 seconds.

So tl;dr, I think the inconsistency is called by functions calls to RFID
sensor read. I haven't verified.
